addons/l10n_in/models/res_config_settings.py

```diff
@@ -8,3 +8,4 @@ class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
     group_l10n_in_reseller = fields.Boolean(implied_group='l10n_in.group_l10n_in_reseller', string="Manage Reseller(E-Commerce)")
+    module_l10n_in_edi = fields.Boolean('Indian Electronic Invoicing')
```

Comment:
not sure with stable policy here as it will need an extra field on the res.config.settings object...

Reply:
indeed, you have to remove that field :man_shrugging:

Reply:
@jco-odoo Done